src/app.tsx

```diff
@@ -23,13 +23,13 @@ function App() {
         <AppContainer>
           <Switch>
             <Route exact path="/">
-              <Chatbot />
+              <WelcomePage />
             </Route>
             <Route path="/results/">
               <ResultsPage />
             </Route>
-            <Route path="/welcome/">
-              <WelcomePage />
+            <Route path="/chat/">
```

Comment:
can you update ResultsPage's `back to questionnaire` links to point to this then please?